client/src/components/Message/Message.css

```diff
@@ -0,0 +1,48 @@
+.Message__message-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  box-sizing: border-box;
+  padding: 1rem;
+  width: 26.19rem;
+  height: calc(71px+2rem);
+  font-family: var(--paragraph-font);
+}
+
+.Message__profilePics {
```

Comment:
Can you fix this so it's inline with the naming convention?

Reply:
Hopefully yes! Now named `.InboxUserProfilesGroup__profile-picture` in separate component